common/src/main/java/com/linecorp/centraldogma/internal/Jackson.java

```diff
@@ -281,6 +295,66 @@ public static String escapeText(String text) {
         return new String(enc.quoteAsString(text));
     }
 
+    public static JsonNode mergeJsonNodes(JsonNode... jsonNodes) {
+        return mergeJsonNodes(ImmutableList.copyOf(requireNonNull(jsonNodes, "jsonNodes")));
+    }
+
+    public static JsonNode mergeJsonNodes(Iterable<JsonNode> jsonNodes) {
+        requireNonNull(jsonNodes, "jsonNodes");
+        final int size = Iterables.size(jsonNodes);
+        checkArgument(size > 0, "jsonNodes size: %s (expected: > 0)", size);
+        final JsonNode first = Iterables.get(jsonNodes, 0);
+        JsonNode merged = first.deepCopy();
+
+        StringBuilder builder = new StringBuilder("/");
+        for (int i = 1; i < size; i++) {
+            final JsonNode addition = Iterables.get(jsonNodes, i);
```

Comment:
This can be seriously inefficient. You can:

```java
final Iterable<JsonNode> it = jsonNodes.iterator();
final JsonNode first = it.next();
JsonNode merged = first.deepCopy();

while (it.hasNext()) {
    final JsonNode addition = it.next();
    ...
}
```